src/oc/storage/api/boards.clj

```diff
@@ -125,13 +126,14 @@
 
 ;; ----- Actions -----
 
-(defn- create-board [conn {new-board :new-board} org-slug]
+(defn- create-board [conn {new-board :new-board user :user} org-slug]
   (timbre/info "Creating board for org:" org-slug)
   (if-let [board-result (board-res/create-board! conn new-board)] ; Add the board
     
     (do
       (timbre/info "Created board:" (:uuid board-result) "for org:" org-slug)
-      (change/send-trigger! (change/->trigger :add :board board-result))
+      (change/send-trigger! (change/->trigger :add board-result))
```

Comment:
What happens when there is a failure with `change/send-trigger!`  Will there be a case when the notification is sent but the change server doesn't process the request successfully? 

My overall question is how do we handle  service request dependencies? Is it better for the change service to send the requests to the notification service or have the storage service handle both as is the case here?

Reply:
To answer the specific question you asked, it depends on where the failure occurs, but if it happens on the consumer side of things then there is a DLQ hooked up to the change services SQS queue, so it'll end up there. We've yet to have anything in production end up in a DLQ (knock on wood!) but at some point would be good to setup some Cloud Alerts on those DLQ queues so we get a notice if they do get any messages.

If the failure happens in storage services before the message is queued? Well presumably we'll get a Sentry for it.

Also note, this is sort of a phased refactor here @thepug 

What I'd like to do is have a fan out from the single SNS notification to the change service as an SQS subscriber to that SNS topic, and then any other SNS subscribers people want to use. So that we don't have Storage services posting both SQS requests to Change services and SNS posts to the topic.

Just didn't want to do this change all at once. Mainly because right now we don't have the SQS or SNS stuff here in the Ansible provisioning. It's been setup manually. Not ideal. We'll have to address that soon.

Reply:
Also didn't do the full refactor (SNS only vs. SNS and SQS both) because the payload we're sending to SNS is quite a bit different than what we send to SQS for Change service. So they'll be a bit of changing on that end too to consume the much more generic (and complete) SNS payload.